src/main/scala/org/bblfsh/client/libuast/alloclist.c

```diff
@@ -0,0 +1,38 @@
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#include "alloclist.h"
+
+#include <jni.h>
+
+#include <stdlib.h>
+
+void initAllocList(AllocList *l, size_t initialSize) {
```

Comment:
Calling this a list when it's actually a vector looks weird.

Reply:
Named it list in the generic sense, but I'll rename it to vector.